search/src/main/kotlin/com/alfresco/content/search/ListViewFilterChips.kt

```diff
@@ -54,12 +54,20 @@ class ListViewFilterChips @JvmOverloads constructor(
     }
 
     private fun String.wrapWithLimit(limit: Int, delimiter: String? = null, multipleValue: Boolean = false): String {
-        if (this.length <= limit)
+        if (this.length <= limit && delimiter == null)
             return this
 
-        if (delimiter != null && this.contains(delimiter)) {
-            val splitStringArray = this.split(delimiter)
-            return context.getString(R.string.name_truncate_in_end, splitStringArray[0].wrapWithLimit(20, ",", true), splitStringArray.size.minus(1))
+        if (delimiter != null) {
+            if (this.contains(delimiter)) {
+                val splitStringArray = this.split(delimiter)
+                val chip1stString = splitStringArray[0]
+                if (chip1stString.length > limit) {
+                    return context.getString(R.string.name_truncate_in_end, chip1stString.wrapWithLimit(20, ",", true), splitStringArray.size.minus(1))
```

Comment:
Limit 20 should be kept in a variable so that it can be changed easily whenever needed.

Reply:
Done